vendor.conf

```diff
@@ -19,7 +19,7 @@ github.com/sirupsen/logrus v1.0.0
 github.com/containers/image 93bced01015eb94bec4821df1876314be8197680
 github.com/docker/docker-credential-helpers d68f9aeca33f5fd3f08eeae5e9d175edf4e731d1
 github.com/ostreedev/ostree-go master
-github.com/containers/storage v1.11
+github.com/containers/storage 06b6c2e4cf254f5922a79da058c94ac2a65bb92f
```

Comment:
Why not moving to the lastest storage v1.12.x? That's already in master and seems to work.